src/jsxstyle-components.js

```diff
@@ -0,0 +1,9 @@
+// Mostly copied from:
+// https://github.com/threepointone/glamor/blob/
+// 5e7d988211330b8e2fca5bb8da78e35051444efd/src/jsxstyle.js
+/*
+ * This is used to generate the helper functions
+ * like: glamorous.block and the <glamorous.Block /> elements
+ */
+
+export default ['view', 'block', 'inlineBlock', 'flex', 'row', 'column']
```

Comment:
if we weren't using rollup, I'd prefer to have this in the same file to save on module closures, but thanks to rollup's module hoisting we should be good here! 🎉